en_US/messages.json

```
@@ -78,5 +78,35 @@
     "qualityEstimationMessage": {
       "message": "Enable quality estimation",
       "description":"Checkbox to enable the quality estimation of the translation."
+    },
+    "formtranslationsReady": {
+      "message": "Ready",
+      "description":"Notifies that the form translation is ready to be used."
+    },
+    "formtranslationsTyping": {
+      "message": "Typing",
+      "description":"Notifies the user that typing is occuring in the form translations area."
+    },
+    "formtranslationsTranslationInProgress": {
+      "message": "Translation in progress…",
+      "description":"Notifies the user that the translation of the input text is happening."
+    },
+    "formtranslationsTranslationCompleted": {
+      "message": "Translation completed.",
```

```suggestion
      "message": "Translation completed",
```

I'd drop the period here. Alternatively, we probably want a period on "Ready" for consistency.